src/main/resources/brapi/sql/R__species.sql

```diff
@@ -0,0 +1,34 @@
+-- See the NOTICE file distributed with this work for additional information
+-- regarding copyright ownership.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+INSERT INTO crop (auth_user_id, id, crop_name) VALUES ('anonymousUser', '4', 'Blueberry') ON CONFLICT DO NOTHING;
```

Comment:
Is the id starting at 4 due to existing data?

Reply:
Exactly, in some run configurations, [this file](https://github.com/Breeding-Insight/brapi-Java-TestServer/blob/develop/src/main/resources/db/sql/R__init_data_01_crops.sql) will be executed by the brapi-server on startup, so starting at 4 avoids collisions.

Reply:
FYI we use a bind mount to overwrite the seed data including that file, that collision is really an edge case.